library/core/src/main/java/com/google/android/exoplayer2/mediacodec/MediaCodecRenderer.java

```diff
@@ -464,6 +467,14 @@ protected void onPositionReset(long positionUs, boolean joining) throws ExoPlayb
     }
   }
 
+  @Override
+  protected void onOperatingRateChanged(float operatingRate) {
+    codecOperatingRate = operatingRate;
+    if (codec != null && format != null) {
```

Comment:
I think just checking the codec is sufficient. There cannot exist a codec if format is null (I think).